### ISSUE TITLE
[r] Out-of-memory issues on `cellxgene_census` R acceptance tests

### ISSUE THREAD

**Issue by johnkerl** [single-cell-data/TileDB-SOMA#2463]
This may, of course, be a bug within `cellxgene_census`. However, the Python and R APIs should be working similarly enough that if one API is not OOMing and another is OOMing, on the same dataset and query pattern, there is likely either a TileDB-SOMA code flaw or a TileDB-SOMA documentation flaw.

Reported by @ebezzi.

**Comment by johnkerl** [comment]
@ebezzi were these resolved by adding `rm` and `gc` calls to your R test cases?

**Comment by ebezzi** [comment]
Yeah, that specific test was fixed via rm+gc. There is still a test that _sometimes_ fails because of OOM (`test_seurat_common-cell-type-large-buffer-size`), and it's not a garbage collection issue since this test loads everything in memory. We'll monitor and see how frequent the failures are, and worst case we'll lower the limit of queried cells (right now it's 15M).
